src/utils.js

```diff
@@ -8,36 +10,50 @@ export const forceNumber = function(n) {
     return n;
 };
 
-export const createHistogramArray = (dist) => {
-    const NO_OF_BINS = 14;
-    let min = Math.min(...dist);
-    let max = Math.max(...dist);
-    let bin_size = (max - min) / NO_OF_BINS;
+export const integerDivision = function(a, b) {
+    let dividend = a / b;
```

Comment:
This function isn't being used, and can probably be removed. If you are going to use this, though, I would add a check before this line that returns null if b is 0, to avoid a divide-by-zero error.

Reply:
Yep, I'll kill it.